tests/functional/object/mcg/ui/test_s3_vector_ui.py

```
@@ -649,3 +652,206 @@ def teardown():
+        )
+        logger.info("Step 10: Updated policy verified — actions: %s", updated_actions)
+
+        # Step 11: Delete the bucket policy via UI.
+        logger.info("Step 11: Deleting bucket policy via UI")
+        perms_ui.delete_bucket_policy_ui(bucket_name=noobaa_bucket_name)
+
+        # Step 12: Poll until the policy is absent via S3 API — the backend may
+        # take a moment to propagate the UI deletion.
+        logger.info("Step 12: Verifying policy is deleted via S3 API")
+
+        def _policy_still_exists():
```

Ruff (0.15.18)

[warning] Missing return type annotation for private function `_policy_still_exists`

Add return type annotation: `bool | None`

(ANN202)